lib/Service/ParticipantService.php

```diff
@@ -161,12 +166,32 @@ public function joinRoom(Room $room, IUser $user, string $password, bool $passed
 				throw new InvalidPasswordException('Provided password is invalid');
 			}
 
-			// User joining a public room, without being invited
-			$this->addUsers($room, [[
-				'actorType' => Attendee::ACTOR_USERS,
-				'actorId' => $user->getUID(),
-				'participantType' => Participant::USER_SELF_JOINED,
-			]]);
+			// User joining a group call through listing
+			if (
+				$room->getType() === Room::GROUP_CALL && (
```

Comment:
or public

Reply:
public is already handled below, I've left it to "self joined" but still need to test whether it works that way or makes sense

Reply:
When you click on a listed conversation you should always be fully added